libavfilter/vf_dnn_detect.c

```diff
@@ -62,29 +62,30 @@ static const AVOption dnn_detect_options[] = {
 
 AVFILTER_DEFINE_CLASS(dnn_detect);
 
-static int dnn_detect_post_proc(AVFrame *frame, DNNData *output, uint32_t nb, AVFilterContext *filter_ctx)
+static int parse_output_to_bounding_box(AVFilterContext *filter_ctx, AVFrame *frame,
+                                        const int position_size,     const int other_size,
+                                        const float *conf_p,         const float *label_id_p,
+                                        const float *position_p,     const int proposal_count,
+                                        const int x0_offset,         const int y0_offset,
+                                        const int x1_offset,         const int y1_offset)
```

Comment:
parse_output_to_bounding_box is designed as a general function, but it would be very complex when more models are supported, for example, some model might not output confidence, etc.

so, we can rename the original dnn_detect_post_proc to dnn_detect_post_proc_ov, and add a new function dnn_detect_post_proc_tf. Keep them separated.

Reply:
Oh, you're right, but these two func have so many common codes, maybe I should use a macro? Or just leave it?

Reply:
we can just leave it unless there's other comment

Reply:
OK